web/modules/custom/hoeringsportal_citizen_proposal/src/Helper/Helper.php

```diff
@@ -255,12 +255,15 @@ public function findOverdueProposals() {
         ->condition('status', 1, '=')
         ->condition('field_content_state', 'active', '=')
         ->condition('field_vote_end', $now->format(DateTimeItemInterface::DATE_STORAGE_FORMAT), '<')
+        ->accessCheck(FALSE)
         ->execute();
```

Comment:
We should check for node type (bundle) in this query. We know that only `citizen_proposal` has the `field_content_state` and `field_vote_end` fields *for now*, but we should make sure that we don't suddenly process other node types.

Reply:
Added type check